scripts/clean-build-dir.js

```diff
@@ -7,19 +13,6 @@ const glob = require('glob');
 const SRC_DIR = 'src';
 const BUILD_DIR = 'build';
 
-if (process.argv[2] === 'clean') {
-  clean();
-} else {
-  // clean before build just to remove any files that may
-  // have been removed from src/ code but are still in build/
-  // from a previous build.
-  clean();
-  // TODO: formerly we were doing other stuff here,
-  // and we might again in the future with work on
-  // https://github.com/natcap/invest-workbench/issues/146
-  // So we should clean this up as part of that issue.
-}
-
```

Comment:
Whoops just noticed a glaring error here. I removed the function call completely.